usersession/agent/rest_api.go

```diff
@@ -334,3 +362,175 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 	}
 	return SyncResponse(nil)
 }
+
+func tryNotifyRefreshToSnapDesktopIntegration(snapName string, operation NotifyRefreshOperation) error {
```

Comment:
```suggestion
func notifyRefreshToSnapDesktopIntegration(snapName string, operation NotifyRefreshOperation) error {
```

Reply:
There is a reason to call it "try": because it's not guaranteed that snapd-desktop-integration is installed; in that case, the code will fallback into regular notifications. But if you still consider that removing it is better, I'll do it.

Reply:
It doesn't matter, because the function returns an error if it doesn't work. It's not the function that 'tries', but rather the caller that does a best effort. So instead it's the caller that should document that we do a best effort attempt at notifying, otherwise we just log an error

Reply:
Changing then.